docs/tutorials/thinking_in_jax.md

```diff
@@ -0,0 +1,393 @@
+# How to think in JAX
+
+In this tutorial you will learn about how JAX operates, so that you can use it more effectively. JAX provides a simple and powerful API for writing accelerated numerical code, and working effectively in JAX sometimes requires extra consideration. This document will help you build a ground-up understanding of the JAX API.
+
+
+## JAX versus NumPy
```

Comment:
As we chatted about, this may be duplicative of the quickstart content. Should we get that in first before we decide what content is appropriate here?

Reply:
TODO: next iteration (your version) :+1: as discussed